src/test/memory-leak/tests/sliderScroll.js

```diff
@@ -0,0 +1,38 @@
+const { loadEnvConfig } = require('@next/env');
+
+const projectDir = process.cwd();
+loadEnvConfig(projectDir);
+
+async function swipeSlider(page, direction = 'left') {
+  const slider = await page.$('.swiper-wrapper');
+  const boundingBox = await slider.boundingBox();
+
+  const startX = direction === 'left' ? boundingBox.x + boundingBox.width - 10 : boundingBox.x + 10;
```

Comment:
need to move calculations into separate function, this makes function more complicated